content/en/blogs/kubernetes-fundamentals-part-1.md

```diff
@@ -0,0 +1,90 @@
+# Kubernetes Fundamentals Part one: A Visual Quick Start</h1>
```

Comment:
Remove `<h1>` tag from here. Please add the header on the top of this blog. Take a look at this example: https://github.com/kubesphere/website/blob/master/content/en/blogs/aws-kubernetes.md
![image](https://user-images.githubusercontent.com/40452856/135955917-ea714e24-4deb-4a8a-a976-70c3381e6ef2.png)